### PR TITLE
Manage io.swagger core and annotations and deprecate com.wordnik

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@pavolloffay could you please review? This is related to https://github.com/hawkular/hawkular-commons/pull/60 The idea behind the present PR is that we should manage `io.swagger` `core` and `annotations` here in parent, deprecate `com.wordnik` now and eventually remove it later.

**Comment by ppalaga** [comment]
@lucasponce could you please review instead of @pavolloffay that does not respond?

File: pom.xml [hunk 1]
```diff
@@ -236,8 +236,10 @@
     <version.com.fasterxml.jackson.core>2.5.4</version.com.fasterxml.jackson.core>
     <!-- keep in sync with modules/system/layers/base/com/google/guava of the current WildFly/EAP -->
     <version.com.google.guava>18.0</version.com.google.guava>
+    <!-- com.wordnik is deprecated. Use io.swagger instead. -->
     <version.com.wordnik.swagger>1.3.13</version.com.wordnik.swagger>
     <version.io.gatling.highcharts>2.1.6</version.io.gatling.highcharts>
+    <version.io.swagger>1.5.8</version.io.swagger>
```
Comment (lucasponce) [inline pom.xml:242]: In alerts I was using 1.5.7 and I followed Inventory version as well.

**Comment by lucasponce** [comment]
Apart of minor version of package, I guess it looks like correct.

File: pom.xml (hunk 1 above)
Comment (ppalaga) [inline pom.xml:242]: Micro upgrades should be fine. But I admit I have not tested with 1.5.8. Anyway, if there are problems with 1.5.8, one can always override `<version.io.swagger>1.5.8</version.io.swagger>` in the child repo.

**Comment by pavolloffay** [comment]
I looks good to me :+1:

**Comment by pavolloffay** [comment]
inventory works fine with `<version.io.swagger>1.5.8</version.io.swagger>`

**Pull request merged**